test/uk/gov/hmrc/gform/services/FormatValidationSpec.scala

```diff
@@ -166,4 +166,68 @@ class FormatValidationSpec extends Spec {
     result.toEither should beLeft(Map(fieldValue.id -> Set("must be a whole number")))
   }
 
+  "UTR" should "be valid "in {
```

Comment:
How about extracting common part of these tests into trait? They all look very similar.